widget/embedded/src/components/Filter/FilterContent.tsx

```diff
@@ -0,0 +1,75 @@
+import type { PropTypes } from './Filter.type';
+
+import { i18n } from '@lingui/core';
+import {
+  Button,
+  Divider,
+  ListItemButton,
+  Radio,
+  RadioRoot,
+  Typography,
+} from '@rango-dev/ui';
+import { TransactionStatus } from 'rango-types';
+import React from 'react';
+
+import {
+  centeredFlexContainer,
+  FilterContainer,
+  FilterList,
+} from './Filter.style';
+
+const list = [
+  {
+    id: TransactionStatus.SUCCESS,
+    title: i18n.t('Complete'),
+    hasDivider: true,
+  },
+  {
+    id: TransactionStatus.RUNNING,
+    title: i18n.t('Running'),
+    hasDivider: true,
+  },
+  { id: TransactionStatus.FAILED, title: i18n.t('Failed'), hasDivider: false },
+];
+export function FilterContent(props: PropTypes) {
+  const { filterBy: value, onClickItem } = props;
+  return (
+    <FilterContainer>
+      <div className={centeredFlexContainer()}>
+        <Typography size="small" variant="body">
+          {i18n.t('Status')}
+        </Typography>
+        <Button variant="ghost" size="xsmall" onClick={() => onClickItem('')}>
+          {i18n.t('Reset')}
+        </Button>
+      </div>
+      <Divider size={10} />
+      <FilterList>
+        {list.map((item) => {
+          return (
+            <React.Fragment key={item.id}>
+              <ListItemButton
+                style={{ height: '40px', width: '100%' }}
+                selected={false}
+                hasDivider={item.hasDivider}
+                id={item.id}
+                start={
+                  <div className={centeredFlexContainer()}>
+                    <RadioRoot value={value}>
```

Comment:
RadioRoot is a component to containt all Radio components. it is missused.

Reply:
It's not fixed. please change it to sth like this:

```
      <RadioRoot value={value}>
        <FilterList>
          {props.list.map((item, index) => (
            <ListItemButton
              key={item.id}
              style={{ height: '40px', width: '100%' }}
              selected={false}
              hasDivider={props.list.length - 1 != index}
              id={item.id}
              title={
                <Typography size="medium" variant="body">
                  {item.title}
                </Typography>
              }
              start={<Radio value={item.id} />}
              onClick={onClickItem}
            />
          ))}
        </FilterList>
      </RadioRoot>
```

Reply:
Sorry, You're right. Fixed it